README.md

```diff
@@ -163,16 +164,30 @@ bukkit {
 <summary><strong>Groovy</strong></summary>
 
 ```groovy
+import net.minecrell.pluginyml.bukkit.BukkitPluginDescription
+import net.minecrell.pluginyml.paper.PaperPluginDescription
+
 plugins {
-    id 'net.minecrell.plugin-yml.paper' version '0.6.0-SNAPSHOT'
+    id 'java'
+    id 'net.minecrell.plugin-yml.paper' version '0.5.3'
```

Comment:
Paper doesn't exist in 0.5.3 yet and will only be in 0.6.0 so please keep this as-is for now. I'll update it when releasing 0.6.0 :)

Reply:
In the parent commit (after you merged #30), some of the version number examples had been updated, and others hadn't, so I figured you'd update them all at once for the version bump.

Reply:
The idea was to only use `0.6.0-SNAPSHOT` to the paper examples since they explicitly require 0.6.0. But you're right that it's somewhat inconsistent because the Bukkit examples also contain foliaSupported which is not in 0.5.3 either.